core/base/document.php

```diff
@@ -1619,6 +1652,14 @@ protected function get_have_a_look_url() {
 		return $this->get_permalink();
 	}
 
+	protected function get_main_dashboard_url() {
+		$url = get_dashboard_url();
+
+		$url = apply_filters( 'elementor/document/urls/main_dashboard', $url, $this );
```

Comment:
Please add a PHPDoc for this filter
See example: https://github.com/elementor/elementor/blob/134b50fd37496a4017f20235c1313f2537f496c5/core/documents-manager.php#L554